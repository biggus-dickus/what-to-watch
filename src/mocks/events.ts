// JSDom has no implementation for common methods of HTML media elements.
// Tests will fail without this. Sad but true.
// https://github.com/jsdom/jsdom/issues/2155#issuecomment-366703395
export const mockHTMLMediaMethods = (): void => {
  window[`HTMLMediaElement`].prototype.load = () => { /* do nothing */ };
  window[`HTMLMediaElement`].prototype.play = () => { /* do nothing */ };
  window[`HTMLMediaElement`].prototype.pause = () => { /* do nothing */ };
};
