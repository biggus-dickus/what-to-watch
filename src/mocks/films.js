export const films = [
  {
    name: `Fantastic Beasts: The Crimes of Grindelwald`,
    picUrl: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Bohemian Rhapsody`,
    picUrl: `img/bohemian-rhapsody.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Macbeth`,
    picUrl: `img/macbeth.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Aviator`,
    picUrl: `img/aviator.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `We need to talk about Kevin`,
    picUrl: `img/we-need-to-talk-about-kevin.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `What We Do in the Shadows`,
    picUrl: `img/what-we-do-in-the-shadows.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Revenant`,
    picUrl: `img/revenant.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Johny English`,
    picUrl: `img/johnny-english.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Shutter Island`,
    picUrl: `img/shutter-island.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Pulp Fiction`,
    picUrl: `img/pulp-fiction.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `No Country for Old Men`,
    picUrl: `img/no-country-for-old-men.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Snatch`,
    picUrl: `img/snatch.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Moonrise Kingdom`,
    picUrl: `img/moonrise-kingdom.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Seven Years in Tibet`,
    picUrl: `img/seven-years-in-tibet.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Midnight Special`,
    picUrl: `img/midnight-special.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `War of the Worlds`,
    picUrl: `img/war-of-the-worlds.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Dardjeeling Limited`,
    picUrl: `img/dardjeeling-limited.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Orlando`,
    picUrl: `img/orlando.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Mindhunter`,
    picUrl: `img/mindhunter.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
];
