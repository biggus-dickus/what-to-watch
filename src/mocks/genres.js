export const Genre = {
  ALL: `All genres`,
  COMEDIES: `Comedies`,
  CRIME: `Crime`,
  DOCUMENTARY: `Documentary`,
  DRAMAS: `Dramas`,
  HORROR: `Horror`,
  FAMILY: `Kids & Family`,
  ROMANCE: `Romance`,
  SCI_FI: `Sci-Fi`,
  THRILLERS: `Thrillers`
};
