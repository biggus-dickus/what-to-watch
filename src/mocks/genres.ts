import {Genre as GenreType} from '../types'; // eslint-disable-line

export const Genre = {
  ALL: `All genres`,
  COMEDIES: `Comedies`,
  CRIME: `Crime`,
  DOCUMENTARY: `Documentary`,
  DRAMAS: `Dramas`,
  HORROR: `Horror`,
  FAMILY: `Kids & Family`,
  ROMANCE: `Romance`,
  SCI_FI: `Sci-Fi`,
  THRILLERS: `Thrillers`
};

export const mockGenres = Object.values(Genre);

export const mockGenreEntity: GenreType = {
  currentGenre: Genre.CRIME,
  genres: [...mockGenres],
  onGenreChange: jest.fn()
};
