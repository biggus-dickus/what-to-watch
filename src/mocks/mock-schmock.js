export const mockGenres = [`All`, `BDSM`, `Gay gymnastics`, `Psycho-dramas for aestheticizing faggots`];

export const mockFilms = [
  {
    name: `L'elephant vert`,
    genre: [mockGenres[1], mockGenres[2]],
    pageUrl: `https://www.imdb.com/title/tt1003080/`,
    picUrl: `http://lurkmore.so/images/4/47/Slonik_macro.jpg`,
    videoUrl: `https://storage.googleapis.com/coverr-main/mp4%2FCam-Man.mp4`
  },
  {
    name: `Sal√≤ o le 120 giornate di Sodoma`,
    genre: [mockGenres[3]],
    pageUrl: `https://www.imdb.com/title/tt0073650/`,
    picUrl: `https://m.media-amazon.com/images/M/MV5BYjE0NTU3MTUtYjE0OC00ZGM1LWIwNzYtNDU2YWI2YWJjZjEwXkEyXkFqcGdeQXVyMTA2ODMzMDU@._V1_SY1000_CR0,0,764,1000_AL_.jpg`,
    videoUrl: `etc/na-zdorovnya.mp4`
  },
  {
    name: `Un Chien Andalou`,
    genre: [mockGenres[3]],
    pageUrl: `https://www.imdb.com/title/tt0020530/`,
    picUrl: `https://m.media-amazon.com/images/M/MV5BMjZlY2RjM2ItZGEwOC00ODQwLWFhZDYtMjE3ODMyYWNiOWNlXkEyXkFqcGdeQXVyNTA4NzY1MzY@._V1_SY1000_CR0,0,706,1000_AL_.jpg`,
    videoUrl: `etc/na-zdorovnya.mp4`
  }
];
