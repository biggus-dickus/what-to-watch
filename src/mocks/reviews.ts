import {Review} from '../types'; // eslint-disable-line

export const mockReviews: Review[] = [
  {
    id: 1,
    user: {
      id: 4,
      name: `Kate Muir`,
    },
    rating: 8.9,
    comment: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
    date: `2019-05-08T14:13:56.569Z`
  },
  {
    id: 2,
    user: {
      id: 13,
      name: `Bill Goodykoontz`,
    },
    rating: 8.0,
    comment: `Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight. “The Grand Budapest Hotel” is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
    date: `2019-06-07T21:12:21.569Z`
  }
];
