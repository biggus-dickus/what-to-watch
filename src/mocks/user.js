const mockUser = {
  id: 1,
  email: `Oliver.conner@gmail.com`,
  name: `Oliver.conner`,
  avatar_url: `img/1.png` // eslint-disable-line
};

export default mockUser;
