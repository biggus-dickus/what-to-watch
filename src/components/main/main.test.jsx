import React from 'react';
import renderer from 'react-test-renderer';

import Main from './main';

import {mockFilms, mockGenres} from '../../mocks/mock-schmock';


it(`renders correctly after relaunch`, () => {
  const tree = renderer.create(<Main films={mockFilms} genres={mockGenres} />).toJSON();
  expect(tree).toMatchSnapshot();
});
