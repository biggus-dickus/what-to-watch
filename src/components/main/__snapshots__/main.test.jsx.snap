// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly after relaunch 1`] = `
Array [
  <section
    className="movie-card"
  >
    <div
      className="movie-card__bg"
    >
      <img
        alt="The Grand Budapest Hotel"
        height="552"
        src={
          Object {
            "process": [Function],
          }
        }
        width="1300"
      />
    </div>
    <h1
      className="visually-hidden"
    >
      What to Watch
    </h1>
    <header
      className="page-header movie-card__head"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="user-block"
      >
        <div
          className="user-block__avatar"
        >
          <img
            alt="User avatar"
            height="63"
            src={
              Object {
                "process": [Function],
              }
            }
            width="63"
          />
        </div>
      </div>
    </header>
    <div
      className="movie-card__wrap"
    >
      <div
        className="movie-card__info"
      >
        <div
          className="movie-card__poster"
        >
          <img
            alt="The Grand Budapest Hotel poster"
            height="327"
            src={
              Object {
                "process": [Function],
              }
            }
            width="218"
          />
        </div>
        <div
          className="movie-card__desc"
        >
          <h2
            className="movie-card__title"
          >
            The Grand Budapest Hotel
          </h2>
          <p
            className="movie-card__meta"
          >
            <span
              className="movie-card__genre"
            >
              Drama
            </span>
            <span
              className="movie-card__year"
            >
              2014
            </span>
          </p>
          <div
            className="movie-card__buttons"
          >
            <button
              className="btn btn--play movie-card__button"
              type="button"
            >
              <svg
                height="19"
                viewBox="0 0 19 19"
                width="19"
              >
                <use
                  xlinkHref="img/sprite/sprite.svg#play-s"
                />
              </svg>
              <span>
                Play
              </span>
            </button>
            <button
              className="btn btn--list movie-card__button"
              type="button"
            >
              <svg
                height="20"
                viewBox="0 0 19 20"
                width="19"
              >
                <use
                  xlinkHref="img/sprite/sprite.svg#add"
                />
              </svg>
              <span>
                My list
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <div
    className="page-content"
  >
    <section
      className="catalog"
    >
      <h2
        className="catalog__title visually-hidden"
      >
        Catalog
      </h2>
      <ul
        className="catalog__genres-list"
      >
        <li
          className="catalog__genres-item catalog__genres-item--active"
        >
          <a
            className="catalog__genres-link"
            href="#"
          >
            All genres
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#"
          >
            Comedies
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#"
          >
            Crime
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#"
          >
            Documentary
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#"
          >
            Dramas
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#"
          >
            Horror
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#"
          >
            Kids & Family
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#"
          >
            Romance
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#"
          >
            Sci-Fi
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#"
          >
            Thrillers
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#"
          >
            Good ol' Porn
          </a>
        </li>
      </ul>
      <div
        className="catalog__movies-list"
      >
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Fantastic Beasts: The Crimes of Grindelwald"
              height="175"
              src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Fantastic Beasts: The Crimes of Grindelwald
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Bohemian Rhapsody"
              height="175"
              src="img/bohemian-rhapsody.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Bohemian Rhapsody
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Macbeth"
              height="175"
              src="img/macbeth.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Macbeth
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Aviator"
              height="175"
              src="img/aviator.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Aviator
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="We need to talk about Kevin"
              height="175"
              src="img/we-need-to-talk-about-kevin.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              We need to talk about Kevin
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="What We Do in the Shadows"
              height="175"
              src="img/what-we-do-in-the-shadows.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              What We Do in the Shadows
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Revenant"
              height="175"
              src="img/revenant.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Revenant
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Johny English"
              height="175"
              src="img/johnny-english.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Johny English
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Shutter Island"
              height="175"
              src="img/shutter-island.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Shutter Island
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Pulp Fiction"
              height="175"
              src="img/pulp-fiction.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Pulp Fiction
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="No Country for Old Men"
              height="175"
              src="img/no-country-for-old-men.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              No Country for Old Men
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Snatch"
              height="175"
              src="img/snatch.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Snatch
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Moonrise Kingdom"
              height="175"
              src="img/moonrise-kingdom.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Moonrise Kingdom
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Seven Years in Tibet"
              height="175"
              src="img/seven-years-in-tibet.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Seven Years in Tibet
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Midnight Special"
              height="175"
              src="img/midnight-special.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Midnight Special
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="War of the Worlds"
              height="175"
              src="img/war-of-the-worlds.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              War of the Worlds
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Dardjeeling Limited"
              height="175"
              src="img/dardjeeling-limited.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Dardjeeling Limited
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Orlando"
              height="175"
              src="img/orlando.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Orlando
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          tabIndex="0"
        >
          <button
            className="small-movie-card__play-btn"
            type="button"
          >
            Play
          </button>
          <div
            className="small-movie-card__image"
          >
            <img
              alt="Mindhunter"
              height="175"
              src="img/mindhunter.jpg"
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="https://thepiratebay.org"
            >
              Mindhunter
            </a>
          </h3>
        </article>
      </div>
      <div
        className="catalog__more"
      >
        <button
          className="catalog__button"
          type="button"
        >
          Show more
        </button>
      </div>
    </section>
    <footer
      className="page-footer"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link logo__link--light"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="copyright"
      >
        <p>
          © 
          2019
           What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>,
]
`;
